resources/logback.test.xml

```diff
@@ -0,0 +1,37 @@
+<configuration>
+    <property name="type" value="${ZIGGURAT_LOG_FORMAT:-text}" />
+
+    <if condition = 'property("type").equals("json")'>
+        <then>
+            <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+                <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+                    <layout class="cambium.logback.json.FlatJsonLayout">
+                        <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter">
+                            <prettyPrint>false</prettyPrint>
+                        </jsonFormatter>
+                        <!-- <context>api</context> -->
+                        <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSS'Z'</timestampFormat>
+                        <timestampFormatTimezoneId>UTC</timestampFormatTimezoneId>
+                        <appendLineSeparator>true</appendLineSeparator>
+                    </layout>
+                </encoder>
+            </appender>
+        </then>
+    </if>
+
+    <if condition = 'property("type").equals("text")'>
+        <then>
+            <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+                <layout class="ch.qos.logback.classic.PatternLayout">
+                    <pattern>
+                        %d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg { %mdc }%n
+                    </pattern>
+                </layout>
+            </appender>
+        </then>
+    </if>
+
+    <root level="error">
```

Comment:
this should be level="info"?

Reply:
by default info makes sense.

Reply:
info will printout kakfa info logs during tests, in the older `log4j2.test.xml` the level was set to `error`